crumbles/src/lib.rs

```diff
@@ -473,13 +467,6 @@ impl MmapInner {
         let start_addr = self.bytes.as_ptr() as usize;
         let page_offset = page_num * PAGE_SIZE;
 
-        if page_offset >= self.bytes.len() {
```

Comment:
Shouldn't you still check the OoB? Maybe using max_page

Reply:
Rust still checks out of bounds accesses to the slice itself, and panics if OoB occurs.